Source/panels/spell_book.cpp

```diff
@@ -130,6 +133,11 @@ void DrawSpellBook(const Surface &out)
 	const int textPaddingTop = 7;
 	for (int i = 1; i < 8; i++) {
 		spell_id sn = SpellPages[sbooktab][i - 1];
+		Rectangle iconArea = { GetPanelPosition(UiPanels::Spell, { 11, 18 }), Size { 37, SpellBookDescription.height * 7 - 5 } };
+		if (iconArea.contains(MousePosition)) {
+			InfoString = fmt::format(fmt::runtime(_("{:s}")), pgettext("spell", spelldescription[sn].sSpellDescription));

```

Comment:
There is no need for fmt::format here:

```suggestion
			InfoString = pgettext("spell", spelldescription[sn].sSpellDescription);
```